omod/src/main/webapp/fragments/field/datetimepicker.gsp

```diff
@@ -11,8 +11,14 @@
 
     def dateStringFormat
     def dateISOFormatted
+    def dateStringFormat2
+    def useTime = config.useTime
 
-    if (config.useTime) {
+    if (useTime instanceof String) {
+        useTime = Boolean.parseBoolean(useTime)
+    }
```

Comment:
Shouldn't this also assign useTime = config.useTime if it *isn't* an instance of a string?

Reply:
> def useTime = config.useTime

Not necessarily, coz  I think the assignment is already done up at around line 15.

What I'm doing here is to make sure `useTime` is a date instance.

Reply:
Whoops, my bad, yes, this looks fine.